INGRID/line_tracing.py

```diff
@@ -276,40 +279,41 @@ def flip_NSEW_lookup(self, xpt_ID):
         self.NSEW_lookup[xpt_ID]['coor'] = temp_dict
 
     def map_xpt(self, xpt, magx, xpt_ID='xpt1', visual=False, verbose=False):
-
+        print("map_xpt")
         if xpt_ID == 'xpt1':
             self.analyze_saddle(xpt, xpt_ID)
             self._set_function('rho', 'cw')
 
             NS_buffer = self.NSEW_lookup[xpt_ID]['coor']
 
             N_sol = solve_ivp(self.function, (0, self.dt), NS_buffer['N'], method='LSODA',
-                first_step=self.first_step, max_step=self.max_step, rtol=1e-13, atol=1e-12).y
+                              first_step=self.first_step, max_step=self.max_step, rtol=1e-13, atol=1e-12).y
             S_sol = solve_ivp(self.function, (0, self.dt), NS_buffer['S'], method='LSODA',
-                first_step=self.first_step, max_step=self.max_step, rtol=1e-13, atol=1e-12).y
+                              first_step=self.first_step, max_step=self.max_step, rtol=1e-13, atol=1e-12).y
 
             N_guess = (N_sol[0][-1], N_sol[1][-1])
             S_guess = (S_sol[0][-1], S_sol[1][-1])
 
             r_bounds = (self.grid.rmin, self.grid.rmax)
             z_bounds = (self.grid.zmin, self.grid.zmax)
             N_minimizer = minimize(self.PsiCostFunc, N_guess, method='L-BFGS-B',
-                jac=self.grid.Gradient, bounds=[r_bounds, z_bounds]).x
+                                   jac=self.grid.Gradient, bounds=[r_bounds, z_bounds]).x
             S_minimizer = minimize(self.PsiCostFunc, S_guess, method='L-BFGS-B',
-                jac=self.grid.Gradient, bounds=[r_bounds, z_bounds]).x
+                                   jac=self.grid.Gradient, bounds=[r_bounds, z_bounds]).x
 
-            if (np.linalg.norm(N_minimizer - magx) >= self.eps):
-                self.flip_NSEW_lookup(xpt_ID)
+            # if (np.linalg.norm(N_minimizer - magx) >= self.eps):
+            # self.flip_NSEW_lookup(xpt_ID)
```

Comment:
Do you remember the motivation for commenting out this block? Was this causing an incorrect classification?

Reply:
Yes. That block created a misclassification of a LSN into a USN (for a very large equilibrium with R0 = 5m)